Data/ByteString/Builder/Prim.hs

```diff
@@ -679,7 +679,7 @@ cstring =
     step !addr !k br@(BufferRange op0@(Ptr op0#) ope)
       | W8# ch == 0 = k br
       | op0 == ope =
-          return $ bufferFull defaultChunkSize op0 (step addr k)
+          return $ bufferFull 1 op0 (step addr k)
```

Comment:
Yes, that would be wrong, because then the driver would be allowed to supply an empty buffer to the continuation `(step addr k)`, which leads to the same `bufferFull` call and so on, with an infinite loop similar to #203.

Reply:
Thanks, @clyring. That makes sense.